PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+# Quality Assurance Checklist
+To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
+- [ ] Is proper power protection given to peripheral devices (zener diodes)?  
+- [ ] Are peripheral units used properly (reading datasheet)?  
+- [ ] Are testing points added at useful places?  
+- [ ] Is there proper short to GND protection at MCU outputs (inline resistors)?  
+- [ ] Do ADC inputs have caps?  
```

Comment:
If this is for filtering, it may be noted that the STM32 ADCs (or at least those of the L432KC) pull variable current depending on the voltage (not fun). So a simple RC filter can have disastrous results on the reading.

An alternative solution is an RC filter tied to an unity gain op amp to supply the current and provide high impedance (at the cost of stability/time delay, depending on the op amp bandwidth and the ADC bandwidth).

This was an issue on the MPPTs and Curve Tracer boards that took Hallock and I weeks to figure out.

Reply:
We also had an issue with high RC delay. I listed many things on this list that encourages the designer to think about things that could have been missed. I will add a note to consider RC delay though.